recipes/smp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "smp" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IntelPython/smp/archive/v{{ version }}.tar.gz
+  sha256: 66fbf6e70a9bdfefadd7d3e98dd3415990d81d629ae333055afeeaabdf3c0761
+
+build:
+  number: 0
+  skip: True       # [not linux]
```

Comment:
Make this `noarch: python`?

Reply:
I tried, but I cannot disable testing on windows and mac, right? this module is not exactly platform independent yet. It works on Linux only.

Reply:
It's fine if tests fail in windows and mac for noarch packages. After a feedstock is made, it will run only on Linux.

Reply:
ok, I'll change it back then

Reply:
Still, I don't understand how the feedstock can run on Linux only if the `skip: True` and `-  noarch: python` are mutual exclusive according to the linter?

Reply:
`noarch: python` packages are built on Linux and can be installed in all platforms. We cant restrict the package to be installed in Linux only like pip, but the package is not meant to be used outside of Linux, so that's okay in my opinion.